Makefile

```diff
@@ -18,15 +18,9 @@ check_convention:
 
 uninstall:
 	-yes | sudo pip uninstall solvent
-	-sudo rm -f /etc/bash_completion.d/solvent.sh
-	sudo rm /usr/bin/solvent
```

Comment:
This should also attempt to remove the script from /usr/local/lib since that's the path which the file is being placed in, on Ubuntu. And test that it works.